packages/block-library/src/navigation/index.php

```diff
@@ -1441,6 +1440,10 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
 		}
 		update_post_meta( $post->ID, '_wp_ignored_hooked_blocks', json_encode( $ignored_hooked_blocks ) );
 	}
+
+	// TODO: wp_update_post() to set the post_content to the updated markup (to include the ignoredHookedBlocks metadata).
```

Comment:
I.e. for anchor blocks _other_ than the mock root Nav block -- in case a block hooks e.g. `after` `core/page-list`, we want the `ignoredHookedBlocks` metadata set on that anchor block, in the markup.

Reply:
Come to think of it, we should ideally cover that case with a unit test somewhere 🤔

Reply:
Done in https://github.com/WordPress/gutenberg/pull/59021/commits/9da0b54ab8e904414fae465d696c5ef1f21a8325

Reply:
Will follow up with unit tests